robottelo/products.py

```diff
@@ -985,23 +971,20 @@ def setup_virtual_machine(
                 repo['label'] for repo in self.custom_repos_info if repo['content-type'] == 'yum'
             ]
 
-        setup_virtual_machine(
-            vm,
+        vm.contenthost_setup(
             self.organization['label'],
-            rh_repos_id=rh_repos_id,
-            repos_label=custom_repos_label,
+            rh_repo_ids=rh_repos_id,
+            repo_labels=custom_repos_label,
```

Comment:
Not sure the future generations will understand this, why not rename custom_repos_label and rh_repos_id on lines 965 and 968?

Reply:
Good question, I only dug so far into looking at the content of some of these functions. I'll take a second look at this pattern.